pylua/client3.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+import websockets
+from struct import *
+
+async def hello():
```

Comment:
@oherasym, please stop using indexes in file names. Plus hello is not to descriptive name. Please rename. I suggest "worker", "sender" or try to imagine something better.